SQL/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace SQL
+{
+ 
+1: create table toys (id int, name varchar(100), price float, quantity int);
+2: insert into toys values (1, 'dragon', 20, 20.00);
```

Comment:
in your create statement you price comes before quantity, but they are reversed in your insert statement.